src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -122,11 +122,10 @@ extension ExposureSubmissionCoordinator {
 	/// and returns an ExposureSubmissionTestResultViewController.
 	/// Option 3: (default) return the ExposureSubmissionIntroViewController.
 	private func getInitialViewController(with result: TestResult? = nil) -> UIViewController {
-		#if UITESTING
-		if ProcessInfo.processInfo.arguments.contains("-negativeResult") {
+		#if DEVELOP
```

Comment:
Can you list `DEVELOP` in COMPILATION_CONDITIONS.md

Reply:
Gnaa… nope! Because this should be `DEBUG` and I'm always confusing this shit. 
Will update this in a moment!